src/builder/modules/Generator_Builder.res

```diff
@@ -401,7 +401,7 @@ let defineText = (model: Model.t, text: string) => {
   }
 }
 
-let fillBackgroundColor = (model: Model.t, fillStyle: string) => {
+let fillBackgroundColor = (model: Model.t, ~color: string) => {
```

Comment:
As per my comment above, let's leave this as a `color` rather than `~color`, and feel free to add another function `fillBackgroundColourWithWhite()` if you like.